doc/user/content/cli/_index.md

```diff
@@ -17,7 +17,7 @@ Flag | Default | Modifies
 [`--disable-telemetry`](#telemetry) | N/A | Disables telemetry reporting.
 [`--experimental`](#experimental-mode) | Disabled | *Dangerous.* Enable experimental features.
 [`--introspection-frequency`](#introspection-sources) | 1s | The frequency at which to update [introspection sources](#introspection-sources).
-[`--metrics_scraping-frequency`](#prometheus-metrics) | 1s | The frequency at which to update [prometheus metrics](#prometheus-metrics).
+[`--metrics-scraping-frequency`](#prometheus-metrics) | 10s | The frequency at which to update [prometheus metrics](#prometheus-metrics).
```

Comment:
I know this is pre-existing, but this description isn't quite right, it's more like:

> How often the `mz_metrics` table scrapes and caches [prometheus metrics](#prometheus-metrics).

Reply:
Changed the wording.